src/types/interfaces/search-options.ts

```
@@ -1,10 +1,17 @@
 // Imports
-import { type DeepKeys } from '../types';
+import { Range, type DeepKeys } from '../types';
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)